NOTIFICATIONS.md

```
@@ -18,7 +18,7 @@ If you want to start using the notification endpoint directly with other service
 
 Custom webhook examples:
 - [Jellyfin](#Jellyfin)
-- [Seerr](#Seerr)
+- [Jellyseerr](#Jellyseerr)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid
Expected: `#jellyseerr`; Actual: `#Jellyseerr`

(MD051, link-fragments)